index.html

```diff
@@ -190,7 +190,7 @@ <h1>Contact me</h1>
       <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</P>
     </div>
     <form action="post">
```

Comment:
- [ ] Kindly create a form with [formspree](https://formspree.io/html) services that generate a unique link that you will use in your `action` attribute so that your form can be submitted.